### PR TITLE
#4459 Clean up stream provider

### PR DESCRIPTION
**Pull request by jaroel**

### CODE REVIEW

File: src/ploneintranet/activitystream/browser/stream_provider.py [hunk 1]
```diff
@@ -90,15 +88,12 @@ def activities(self):
                 i += 1
 
     def _activity_visible(self, activity):
-        if IStatusActivity.providedBy(activity) and self.show_microblog:
+        if IStatusActivity.providedBy(activity):
```
Comment (ale-rt) [inline src/ploneintranet/activitystream/browser/stream_provider.py]: is enough to return IStatusActivity.providedBy(activity)

Probably it is better to remove completely this method and just substitute self._activity_visible with IStatusActivity.providedBy

```
[ale@kenobi ploneintranet]$grep -R _activity_visible src/ploneintranet
src/ploneintranet/activitystream/browser/stream_provider.py:            if self._activity_visible(activity):
src/ploneintranet/activitystream/browser/stream_provider.py:    def _activity_visible(self, activity):
```

File: src/ploneintranet/activitystream/browser/stream_provider.py [hunk 2]
```diff
@@ -90,15 +88,12 @@ def activities(self):
...
             return True
         return False
 
     def _activities_statuses(self):
-        if not self.show_microblog:
-            raise StopIteration()
         container = PLONEINTRANET.microblog
```
Comment (ale-rt) [inline src/ploneintranet/activitystream/browser/stream_provider.py:91]: I think also container is always True

Comment (gyst) [inline src/ploneintranet/activitystream/browser/stream_provider.py:91]: NO! It's the outcome of queryUtility i.e. the actual microblog container tool.

Comment (ale-rt) [inline src/ploneintranet/activitystream/browser/stream_provider.py:91]: Ok... here I can see this:
- https://github.com/ploneintranet/ploneintranet/blob/251c8cf9f1e69c38030b6b6ac2f7c93c86ae1e60/src/ploneintranet/core/integration.py#L25

So I think container is always Truish...

If it always evaluate to True, the "if not container" in the line below is redundant.

**Pull request merged**